crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -851,6 +852,50 @@ impl EntityCommands<'_> {
         self.add(try_insert(bundle))
     }
 
+    /// Adds a dynamic component to the entity.
+    ///
+    /// This will overwrite any previous value(s) of the same component type.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// // Note: this is a fairly contrived example, mostly because most uses of dynamic components are
+    /// // complex and abstract
+    ///
+    /// let my_u64_component: TypedComponentId<u64> = world.init_dynamic_component::<u64>();
+    /// // later in the app, in a system:
+    /// commands.insert_dynamic(my_u64_component, 145);
```

Comment:
Code examples need to compile or CI won't pass. This doesn't import `TypedComponentId`, nor there's a `world` or `commands` in scope. Note that you can hide code to setup them by starting those lines with a `#`.

Same in most other code examples you added.